brave/browser-laptop
# Use v2 promotion endpoints

Resolves #14980

## Submitter Checklist:


## Test Plan:
- run browser with the staging flag
- enable payments
- claim grant
- make sure that color that is selected matches the options
- dnd triangle to the wrong color and make sure that you get a new captcha
- dnd triangle to the correct color and make sure that grant is claimed successfully

## Reviewer Checklist:


Tests
